### ISSUE TITLE
Request for functionality to pull out actual predictions on test examples.

### ISSUE THREAD

**Issue by pmitra01** [thunlp/OpenKE#54]
Hi,
Thanks for the great framework!

Would you consider implementing functionality for returning a handle for the actual score predictions , i.e., predicted tensor elements on the test set? It will then be straightforward to rank the retrieved results and gain insight on how the model performs, on demonstrative examples.  
This is essentially the same operation as your framework already covers while computing the MRR, hits@n, etc. 

At the moment, I am implementing this atop the returned trained embeddings, and then carrying out my analysis. I maintain the mappings between entities/relations and their ids; and then fixing say the head 'h'  and the relation 'r', I compute the corresponding scores for *all* possible tail candidates 't' (subject to type constraints), using their corresponding embeddings. Thereafter, I rank to retrieve the n top scoring results. My motivation behind this exercise is to then field those examples that have the true tail at a high rank, and those examples that perform poorly in their predictions. 

I am, at the moment, also investigating the more proper (elegant) way of pulling out the relevant class methods and variables from your own code from config, models, etc. But it would be wonderful if you already have tested, working functionality of the same that you could share.

**Comment by ShulinCao** [comment]
You can refer to the codes in OpenKE-PyTorch/config/Config.py:
![image](https://user-images.githubusercontent.com/19309481/43752207-b7c6baec-9a32-11e8-8098-c41f522da9c4.png)
